.gitignore

```diff
@@ -3,3 +3,9 @@
 dev.html
 jsdoc.json
 /.idea
+node_modules
+dist
```

Comment:
Good catch. Our bower repo and source repo we use in house are separate.  I am just used to doing this.